addons/website_sale/models/sale_order.py

```diff
@@ -92,7 +92,11 @@ def _cart_update(self, product_id=None, line_id=None, add_qty=0, set_qty=0, **kw
         if not order_line:
             values = self._website_product_id_change(self.id, product_id, qty=1)
             order_line = SaleOrderLineSudo.create(values)
-            order_line._compute_tax_id()
+            try:
+                order_line._compute_tax_id()
+            except ValidationError as e:
+                # The validation error is used in backend to display errors in taxcloud config, but should fail silently in frontend
+                _logger.debug("ValidationError occurs during tax compute. %s" % (e))
```

Comment:
@tivisse , Yes, for the front-End we are not Raising Any Validation Error, so While Compute Tax it's break silently(not break front-end same like delivery).

Thanks